ui/src/App/Main.jsx

```diff
@@ -1,12 +1,12 @@
-import React, { Component, Fragment } from 'react';
+import React, { Fragment } from 'react';
 import { BrowserRouter, Switch } from 'react-router-dom';
 import { object, arrayOf } from 'prop-types';
 import { withStyles } from '@material-ui/core/styles';
```

Comment:
For hooks, I think the material team suggests using https://material-ui.com/styles/api/#makestyles-styles-options-hook. Give that a try and see if it works.

Reply:
This is working fine. I didn't know about this since i use an older version generally.

Reply:
It looks like the issue is coming from the python version. Try giving [this](https://github.com/taskcluster/taskcluster-web/issues/306#issuecomment-446611865) a try.

Reply:
Now, strangely this is failing. `makeStyles` is giving and error although its the same as the docs :
![image](https://user-images.githubusercontent.com/22626144/59163151-98c57a00-8b1a-11e9-9cab-2641af7f5662.png)

If i update the `@material-ui/core` module to version 4.0.2 this error goes away but then the theme context properties doesn't work properly!
For now, i guess this is the only issue remaining. Should i also commit the changes with the updated `@material-ui` module for you to check them out?